### DISCUSSION TITLE
Kimai deacreasing an hour

### DISCUSSION THREAD

**Discussion by abachtold** [kimai/kimai#3153] [Questions & Answers] [answered]
Every time I log a time it decreases 1 hour.
In the dashboard
![image](https://user-images.githubusercontent.com/67599238/153880229-9a724dbf-8cee-4b33-ab01-2af0a2e7cd69.png)

In the task

![image](https://user-images.githubusercontent.com/67599238/153880266-1afdd660-3572-4a8b-91bb-15a57d913f12.png)

It's the same task. I want to put the time and that it'll not change. My timezone is correct.

**Comment by kevinpapst** [comment]
I don't know how that is possible ... please test your configuration in the demo installation and let me know if you see it there as well.  
And please share your entire configuration and user settings.

**Comment by abachtold** [comment, marked as answer]
In the demo I changed language/timezone of the user for the same as mine and it did not change the time. 

![image](https://user-images.githubusercontent.com/67599238/153933260-7b158312-ac77-40bd-862a-2fbbb9aa2a10.png)

![image](https://user-images.githubusercontent.com/67599238/153933317-63d4c3c0-8bc5-48a2-b588-694278c75e32.png)

![image](https://user-images.githubusercontent.com/67599238/153933555-7616ce71-0653-4be5-9ab5-ce7c55ef7b86.png)

![image](https://user-images.githubusercontent.com/67599238/153933611-1305fedd-3776-42c9-a647-4792fd6ef79d.png)

> beberlei/doctrineextensions 	v1.3.0
> behat/transliterator 	v1.3.0
> composer/package-versions-deprecated 	1.11.99.4
> doctrine/annotations 	1.13.2
> doctrine/cache 	2.1.1
> doctrine/collections 	1.6.8
> doctrine/common 	3.2.0
> doctrine/dbal 	2.13.4
> doctrine/deprecations 	v0.5.3
> doctrine/doctrine-bundle 	2.4.3
> doctrine/doctrine-migrations-bundle 	3.2.0
> doctrine/event-manager 	1.1.1
> doctrine/inflector 	2.0.4
> doctrine/instantiator 	1.4.0
> doctrine/lexer 	1.2.1
> doctrine/migrations 	3.3.0
> doctrine/orm 	2.10.2
> doctrine/persistence 	2.2.3
> doctrine/sql-formatter 	1.1.1
> egulias/email-validator 	3.1.2
> erusev/parsedown 	1.7.4
> exsyst/swagger 	v0.4.2
> ezyang/htmlpurifier 	v4.13.0
> friendsofphp/proxy-manager-lts 	v1.0.5
> friendsofsymfony/rest-bundle 	3.1.0
> gedmo/doctrine-extensions 	v3.1.0
> handcraftedinthealps/rest-routing-bundle 	1.0.4
> jms/metadata 	2.5.1
> jms/serializer 	3.15.0
> jms/serializer-bundle 	3.10.0
> kevinpapst/adminlte-bundle 	3.6
> laminas/laminas-code 	3.4.1
> laminas/laminas-escaper 	2.9.0
> laminas/laminas-eventmanager 	3.4.0
> laminas/laminas-zendframework-bridge 	1.4.0
> league/csv 	9.7.2
> league/html-to-markdown 	5.0.1
> lorenzo/pinky 	1.0.5
> maennchen/zipstream-php 	2.1.0
> markbaker/complex 	3.0.1
> markbaker/matrix 	3.0.0
> monolog/monolog 	1.26.1
> mpdf/mpdf 	v8.0.14
> myclabs/deep-copy 	1.10.2
> myclabs/php-enum 	1.8.3
> nelmio/api-doc-bundle 	v3.9.1
> nelmio/cors-bundle 	2.1.1
> onelogin/php-saml 	3.6.1
> pagerfanta/pagerfanta 	v2.7.2
> paragonie/random_compat 	v9.99.100
> phpdocumentor/reflection-common 	2.2.0
> phpdocumentor/reflection-docblock 	5.3.0
> phpdocumentor/type-resolver 	1.5.1
> phpoffice/phpspreadsheet 	1.19.0
> phpoffice/phpword 	0.18.2
> phpstan/phpdoc-parser 	1.2.0
> psr/cache 	1.0.1
> psr/container 	1.1.1
> psr/http-client 	1.0.1
> psr/http-factory 	1.0.1
> psr/http-message 	1.0.1
> psr/log 	1.1.4
> psr/simple-cache 	1.0.1
> robrichards/xmlseclibs 	3.1.1
> sensio/framework-extra-bundle 	v6.2.1
> setasign/fpdi 	v2.3.6
> symfony/amazon-mailer 	v4.4.27
> symfony/asset 	v4.4.27
> symfony/cache 	v4.4.33
> symfony/cache-contracts 	v2.4.0
> symfony/config 	v4.4.33
> symfony/console 	v4.4.33
> symfony/css-selector 	v4.4.27
> symfony/debug 	v4.4.31
> symfony/dependency-injection 	v4.4.33
> symfony/deprecation-contracts 	v2.4.0
> symfony/doctrine-bridge 	v4.4.31
> symfony/dotenv 	v4.4.33
> symfony/error-handler 	v4.4.30
> symfony/event-dispatcher 	v4.4.30
> symfony/event-dispatcher-contracts 	v1.1.9
> symfony/expression-language 	v4.4.30
> symfony/filesystem 	v4.4.27
> symfony/finder 	v4.4.30
> symfony/flex 	v1.17.2
> symfony/form 	v4.4.33
> symfony/framework-bundle 	v4.4.31
> symfony/google-mailer 	v4.4.25
> symfony/http-client 	v4.4.33
> symfony/http-client-contracts 	v2.4.0
> symfony/http-foundation 	v4.4.33
> symfony/http-kernel 	v4.4.33
> symfony/inflector 	v4.4.27
> symfony/intl 	v4.4.31
> symfony/mailchimp-mailer 	v4.4.27
> symfony/mailer 	v4.4.27
> symfony/mailgun-mailer 	v4.4.27
> symfony/mime 	v4.4.31
> symfony/monolog-bridge 	v4.4.27
> symfony/monolog-bundle 	v3.7.0
> symfony/options-resolver 	v4.4.30
> symfony/polyfill-ctype 	v1.23.0
> symfony/polyfill-intl-grapheme 	v1.23.1
> symfony/polyfill-intl-icu 	v1.23.0
> symfony/polyfill-intl-idn 	v1.23.0
> symfony/polyfill-intl-normalizer 	v1.23.0
> symfony/polyfill-php73 	v1.23.0
> symfony/polyfill-php80 	v1.23.1
> symfony/polyfill-php81 	v1.23.0
> symfony/polyfill-uuid 	v1.23.0
> symfony/postmark-mailer 	v4.4.27
> symfony/property-access 	v4.4.30
> symfony/property-info 	v4.4.31
> symfony/routing 	v4.4.30
> symfony/security-bundle 	v4.4.27
> symfony/security-core 	v4.4.33
> symfony/security-csrf 	v4.4.27
> symfony/security-guard 	v4.4.27
> symfony/security-http 	v4.4.30
> symfony/sendgrid-mailer 	v4.4.30
> symfony/serializer 	v4.4.33
> symfony/service-contracts 	v2.4.0
> symfony/stopwatch 	v4.4.27
> symfony/string 	v5.3.10
> symfony/translation 	v4.4.32
> symfony/translation-contracts 	v2.4.0
> symfony/twig-bridge 	v4.4.27
> symfony/twig-bundle 	v4.4.30
> symfony/uid 	v5.3.10
> symfony/validator 	v4.4.33
> symfony/var-dumper 	v4.4.33
> symfony/var-exporter 	v4.4.31
> symfony/webpack-encore-bundle 	v1.12.0
> symfony/yaml 	v4.4.29
> tijsverkoyen/css-to-inline-styles 	2.2.3
> twig/cssinliner-extra 	v3.3.3
> twig/extra-bundle 	v3.3.3
> twig/inky-extra 	v3.3.3
> twig/intl-extra 	v3.3.3
> twig/string-extra 	v3.3.3
> twig/twig 	v3.3.3
> webmozart/assert 	1.10.0
> willdurand/jsonp-callback-validator 	v1.1.0
> willdurand/negotiation 	3.0.0
> zircote/swagger-php 	2.1.0
> 

The user preference

![image](https://user-images.githubusercontent.com/67599238/153935569-980f22a9-8b71-4d15-946e-1a997a4ff5cc.png)

My server timezone

> Local time: Mon 2022-02-14 16:21:16 -03
> Universal time: Mon 2022-02-14 19:21:16 UTC
> RTC time: Mon 2022-02-14 19:21:17
> Time zone: America/Sao_Paulo (-03, -0300)
> System clock synchronized: yes
> NTP service: active
> RTC in local TZ: no


Do you want any other system info? I really don't know from where this configuration is coming. I think it has something with day light saving in the time zone. But I can't find anything on php or my server with that.

**Reply by kevinpapst** [reply]
Yeah, that sounds as if it could be the reason:

> it has something with day light saving in the time zone
 
This data is usually stored in "libicu". You should make an update of your system, eg. "apt-get update && apt-get upgrade".

**Reply by abachtold** [reply]
I already did, it stays the same. As you said of libicu I'm searching how to change it manually. Because all my config is up-to-date and I did not enable any day light saving config.

**Reply by kevinpapst** [reply]
Are you using an outdated PHP version? Maybe updating it can solve the issue. 
What OS / versions are you using?

**Reply by abachtold** [reply]
Debian 10 - Buster
PHP 7.3 fpm

**Reply by kevinpapst** [reply]
Try to upgrade to a higher PHP version, which should bring in an updated icu version (hopefully).

**Reply by abachtold** [reply]
I updated to 8.1 and it worked. Again, thanks for all the help

**Reply by kevinpapst** [reply]
Glad it works... BUT: be careful, Kimai is not officially supporting 8.1

**Reply by abachtold** [reply]
Oh. I did not see that. I'll downgrade to 8.0
